xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/suggest/xwiki.selectize.js

```diff
@@ -203,6 +203,31 @@ define('xwiki-selectize', [
         });
       }
     }
+    let oldOnPositionDropdown = this.selectize.onOptionSelect;
+    this.selectize.onOptionSelect = function(e) {
+      oldOnPositionDropdown.call(this, e);
+      this.get$('dropdown').find('.option').attr('aria-selected','false');
+      // clear selection on all previously selected elements first
+      this.get$('dropdown').find('.selected').attr('aria-selected','true');
+    }
+    let oldSetActiveOption = this.selectize.setActiveOption;
+    this.selectize.setActiveOption = function(option, scroll, animate) {
+      if (this.liveRegion) {
+        if (option instanceof jQuery) {
+          this.liveRegion.text(option.text());
+        } else if (option instanceof HTMLElement) {
+          this.liveRegion.text(option.innerText);
+        } else {
+          this.liveRegion.text("");
+        }
+      }
```

Comment:
```suggestion
      this.liveRegion?.text($(option).text());
```

I think this can be simplified, because ``$($(element))`` behaves the same as ``$(element)`` and ``text()`` returns the empty string if the jQuery collection is empty.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3700/commits/017fe211981215802a8a19a5fbaceca40a321e0f :+1: 

Thanks for the help.
I did forget about optional chaining and I didn't take enough time to look at the bigger picture for jquery/html types.